test/test_events.py

```diff
@@ -202,11 +204,13 @@ def test_UQCS_events_typical(uqcsbot: MockUQCSBot):
     messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
     assert len(messages) == 2
     expected = ("Events in the *next _2_ weeks*\r\n"
-                "*AUG 2 0:00 - AUG 5 23:59* - `CodeNetwork Hackathon` - _River City Labs_\r\n"
-                "*AUG 2 18:00 - 20:00* - `Dr Corey Shou` - _TBC_\r\n"
-                "*AUG 6 18:00 - 20:00* - `vim &amp; tmux - Neil Ashford` - _Hawken 50-T103_\r\n"
-                "*AUG 13 18:00 - 20:00* - `Robogals x UQ Robotics x UQCS Social Event`"
-                + " - _Hawken 50-C207_")
+                "*FRI AUG 2 0:00 - MON AUG 5 23:59* - `CodeNetwork Hackathon` - "
+                "_River City Labs_\r\n"
```

Comment:
Message line continued; begin with `+`.
Move joining characters (` - `) to latter line.